CONTRIBUTING.rst

```diff
@@ -152,220 +133,208 @@ Debian/Ubuntu
 
    -  Generate build files using ``cmake``
 
-      On RedHat Enterprise Linux CentOS 7:
-
-      .. code:: sh
-
-         cmake3 .. -DCMAKE_C_COMPILER=/opt/rh/devtoolset-8/root/usr/bin/gcc  -DCMAKE_CXX_COMPILER=/opt/rh/devtoolset-8/root/usr/bin/g++ -DCMAKE_BUILD_TYPE=Release -DBUILD_ONLY="s3;transfer" -DBUILD_SHARED_LIBS=OFF -DMINIMIZE_SIZE=ON -DCMAKE_INSTALL_PREFIX="${AWS_LIBS}"
-
-      On RedHat Enterprise Linux CentOS 8:
-
       .. code:: sh
 
-         cmake .. -DCMAKE_BUILD_TYPE=Release -DBUILD_ONLY="s3;transfer" -DBUILD_SHARED_LIBS=OFF -DMINIMIZE_SIZE=ON -DCMAKE_INSTALL_PREFIX="${AWS_LIBS}"
+         cmake .. -DCMAKE_BUILD_TYPE=Release '-DBUILD_ONLY=s3;transfer' -DBUILD_SHARED_LIBS=OFF -DMINIMIZE_SIZE=ON -DCMAKE_INSTALL_PREFIX="${AWS_LIBS}"
 
    -  Install the SDK
 
-   .. code:: sh
+      .. code:: sh   
 
-      make install
+         make install
 
-7. Build Percona Server for MongoDB
+6. Build Percona Server for MongoDB
 
    -  Change directory to ``percona-server-mongodb``
 
    .. code:: sh
 
       cd percona-server-mongodb
 
-   -  Build Percona Server for MongoDB from ``buildscripts/scons.py``.
-      On RedHat Enterprise Linux / CentOS 7:
+   - With features included in Percona Supported Builds:
+      
+   .. code:: sh  
 
-   .. code:: sh
+      buildscripts/scons.py --disable-warnings-as-errors --release --ssl --opt=on -j$(nproc --all) --use-sasl-client --wiredtiger --audit --inmemory --hotbackup --enable-fipsmode CPPPATH="${AWS_LIBS}/include" LIBPATH="${AWS_LIBS}/lib ${AWS_LIBS}/lib64" install-mongod install-mongos install-mongo
 
-      python3 buildscripts/scons.py CC=/opt/rh/devtoolset-8/root/usr/bin/gcc CXX=/opt/rh/devtoolset-8/root/usr/bin/g++ -j$(nproc --all) --jlink=2  --disable-warnings-as-errors --ssl --opt=on --use-sasl-client --wiredtiger --audit --inmemory --hotbackup CPPPATH="${AWS_LIBS}/include" LIBPATH="${AWS_LIBS}/lib" install-mongod
+   - Without features included in Percona Supported Builds:
 
-   On RedHat Enterprise Linux / CentOS 8:
+   .. code:: sh   
 
-   .. code:: sh
+   buildscripts/scons.py --disable-warnings-as-errors --release --ssl --opt=on -j$(nproc --all) --use-sasl-client --wiredtiger --audit --inmemory --hotbackup CPPPATH="${AWS_LIBS}/include" LIBPATH="${AWS_LIBS}/lib ${AWS_LIBS}/lib64" install-mongod install-mongos install-mongo
 
-      buildscripts/scons.py -j$(nproc --all) --jlink=2  --disable-warnings-as-errors --ssl --opt=on --use-sasl-client --wiredtiger --audit --inmemory --hotbackup CPPPATH="${AWS_LIBS}/include" LIBPATH="${AWS_LIBS}/lib64" install-mongod
+This command builds the core of the database. Other available targets for the
+``scons`` command are: 
 
-This command builds only the database. Other available targets for the
-``scons`` command are: - ``mongod`` - ``mongos`` - ``mongo`` - ``core``
-(includes ``mongod``, ``mongos``, ``mongo``) - ``all``
+- ``mongod`` 
+- ``mongos`` 
+- ``mongo`` 
+- ``core`` (includes ``mongod``, ``mongos``, ``mongo``) 
+- ``all``
```

Comment:
Should be install-mongod, install-mongos, install-mongo, install-core, install-all (v6.0 and newer)